keyboards/gboards/gergoplex/matrix.c

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
Is a custom matrix really needed?

Reply:
Is there any way to get these suggested changes into their own branch so I can test them without trashing the PR? Need to verify these all work and compile

Reply:
Not sure if Github has a feature like this, but you could commit all the suggestions as one and pull in the PR into new branch on your computer.  
You could check it out, fine-tune it and drop it if it's a disaster.

Feel free to ping me on QMK Discord if the above doesn't make any sense. Words are hard... :D